### ISSUE TITLE
Does not work with Save All

### ISSUE THREAD

**Issue by jibbers42** [SamVerschueren/vscode-final-newline#8]
A newline is not added during a Save All for all unsaved files that are not active. It does work for the active file. I see it intentionally rejects non-active files. I assume that was added to avoid something bad, but I thought I would open a ticket anyway :).

``` js
private _insertFinalNewline() {
    if (!window.activeTextEditor) {
        return;
    }
    ...
}
```

**Comment by SamVerschueren** [comment]
Hi @Jibbers42, thanks for reporting. I never have tested it with the `Save All` action. It was not left out on purpose. But when looking into the API documentation I can't find a decent way of solving this issue at the moment. I opened a ticket in the VSCode issue tracker to add a new API feature that would allow me to solve this problem. Will keep you posted!

**Comment by SamVerschueren** [comment]
Fixed in 5268797

Will be released with the next VS Code release.
